android/KMEA/app/src/main/java/com/tavultesoft/kmea/data/Keyboard.java

```diff
@@ -21,7 +24,27 @@
   private String font;
   private String oskFont;
 
+  // JSON keys
+  public static String KB_NEW_KEYBOARD_KEY = "isNewKeyboard";
+  public static String KB_FONT_KEY = "font";
+  public static String KB_OSK_FONT_KEY = "oskFont";
+
+  /**
+   * Constructor using JSON Objects from installed keyboards list
+   * @param installedObj
+   */
+  public Keyboard(JSONObject installedObj) {
+    super(installedObj);
```

Comment:
This line isn't needed because `this.fromJSON` does the heavy lifting and calls its super directly anyway. The superclass is an abstract class; it will never be instantiated so we can optimise away empty constructors etc.